supervisor/ipsetsupervisor_test.go

```
@@ -13,9 +14,10 @@ func doNewIPSetSupervisor(t *testing.T) *ipsetSupervisor {
 	pe := mockenforcerDefaultFQConfig(t)
 	ipt := provider.NewTestIptablesProvider()
 	ips := provider.NewTestIpsetProvider()
+	ipu, _ := iptablesutils.NewIpsetUtils(ipt, ips)
```

Don't ignore the error here